storage/StorageExample/Shared/ContentView.swift

```diff
@@ -229,13 +271,25 @@ struct OrangeButton: ButtonStyle {
 }
 
 extension UIImage {
-  var jpeg: Data? { jpegData(compressionQuality: 1) }
+  var jpeg: Data? {
+    #if os(iOS)
+      jpegData(compressionQuality: 1)
+    #elseif os(macOS)
+      let cgImage = self.cgImage(forProposedRect: nil, context: nil, hints: nil)!
+      let bitmapRep = NSBitmapImageRep(cgImage: cgImage)
+      let jpegData = bitmapRep.representation(
+        using: NSBitmapImageRep.FileType.jpeg,
+        properties: [:]
+      )!
+      return jpegData
```

Comment:
Instead of force-unwrapping in this method, you can just return `nil` since the return type is optional.